### ISSUE TITLE
Documentation could be clearer around module resolution

### ISSUE THREAD

**Issue by paulpopus** [google/closure-compiler-npm#181]
Continuing the discussion from https://groups.google.com/g/closure-compiler-discuss/c/XjXfIQlCnfk

**TL;DR** of that thread  
I found the documentation around the Closure-Compiler to be slightly confusing about node_module resolution, the compiler is very advanced so I assumed it will handle node_module resolution automatically and technically it can actually handle JS module resolution to some extent, however this wasn't clear from the documentation at all.

I think the documentation on https://github.com/google/closure-compiler-npm/blob/master/packages/google-closure-compiler/docs/gulp.md should be updated to specifically mention _node_module_ resolution and the options the developer has with some examples.

I'll open a PR with this soon.

**Comment by paulpopus** [comment]
PR created https://github.com/google/closure-compiler-npm/pull/182

**Comment by ChadKillingsworth** [comment]
We've updated the documentation on the main compiler wiki.
